data/json/requirements/toolsets.json

```diff
@@ -1,5 +1,23 @@
 [
   {
+    "id": "cooking_standard",
+    "type": "requirement",
+    "//": "Cooking of standard meals",
+    "qualities": [ { "id": "COOK", "level": 2 } ],
+    "tools": [ [
+      [ "fire", -1 ],
+      [ "hotplate", 10 ],
+      [ "mess_kit", 10 ],
+      [ "mil_mess_kit", 10 ],
+      [ "oil_cooker", 10 ],
+      [ "survivor_mess_kit", 10 ],
+      [ "gasoline_cooker", 10 ],
+      [ "charcoal_cooker", 10 ],
+      [ "popcan_stove", 10 ],
+      [ "toolset", 10 ]
+    ] ]
+  },
```

Comment:
This isn't really a good example:
* It is not granular enough, it could easily have 1 charge of heat items rather than 10
* The heat sources should be their own requirement, to actually fix the problem of having to repeat the tools rather than just mitigating it partially
* The quality shouldn't be bundled here, otherwise we'll need a separate copy of this requirement for easier to cook items

It would be easier to manage if requirements generally had only the tools for the given job. Welding requires the goggles, but heating up in general doesn't require a frying pan.